packages/atomic/src/components/atomic-tab/atomic-tab.tsx

```diff
@@ -50,12 +53,16 @@ export class AtomicTab {
   }
 
   render() {
+    const activeClass = this.state.isActive ? 'active' : 'btn-outline-primary';
     return (
-      <button class="tab" onClick={() => this.handleClick()}>
-        <span class={this.state.isActive ? 'active' : ''}>
+      <nav class="nav nav-pills">
```

Comment:
I'm not sure about the `nav` tag (maybe an other tag), but the button needs a wrapper to leverage bootstrap classes. For example: 
```
.nav-pills {
  .nav-link {
    @include border-radius($nav-pills-border-radius);
  }
```

I tried using only a `button` but it needed additional css to handle the color when hovering.

Reply:
Hmm nav is supposed to be a set of navigation links, a section https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav
I guess then it shouldn't be 1 nav per tab? But a whole nav for the section. In this case it can be the implementer's choice to add one in his markup or not. Or we could create a atomic-tabs component that would take multiple atomic-tab as slot